src/test/java/com/squareup/kotlinpoet/AnnotationSpecTest.kt

```diff
@@ -256,6 +256,54 @@ class AnnotationSpecTest {
         "@kotlin.Deprecated(\"Nope\", kotlin.ReplaceWith(\"Nope\"))")
   }
 
+  @Test fun annotationStringsAreConstant() {
+    val text = "This is a long string with a newline\nin the middle."
+    val builder = AnnotationSpec.builder(Deprecated::class)
+        .addMember("%S", text)
+
+    assertThat(builder.build().toString()).isEqualTo("" +
+        "@kotlin.Deprecated(\"This is a long string with a newline\\nin the middle.\")")
+  }
+
+  @Test fun literalAnnotation() {
+    val annotationSpec = AnnotationSpec.builder(Suppress::class)
+        .addMember("%S", "Things")
+        .build()
+
+    val file = FileSpec.builder("test", "Test")
+        .addFunction(FunSpec.builder("test")
+            .addStatement("%L", annotationSpec)
+            .addStatement("val annotatedString = %S", "AnnotatedString")
+            .build())
+        .build()
+    assertThat(file.toString().trim()).isEqualTo("""
+      package test
+
+      import kotlin.Suppress
+
+      fun test() {
+          @Suppress("Things")
+          val annotatedString = "AnnotatedString"
+      }
+    """.trimIndent())
```

Comment:
Just for consistency with the rest of the codebase, can you please switch to `trimMargin()`?

Reply:
Done 56e3896